torchbox/torchbox.com
# 1272 | FE setup

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1346361272)

### Description of Changes Made
- Adds Outfit variable font and set the weights between 300 - 600. I can only see 300, 400 and 600 in the design so this should cover those instances. I've also added some notes on this in the docs folder. 
- Also adds Preflight which was missing from the Tailwind config that provides a CSS reset. As a result of this i've removed some of the rules that were set in `_base.scss` as they were duplicated. I've added a comment about this in the file. This has also removed all the font sizes from headings which be resolved in my text ticket.
- Adds a third breakpoint in based on what Ben mentioned when discussing the grid system. The value for this might change after the handover meeting. Also added docs for the breakpoints.
> The grid container will have left/right margin of 120px at large desktop (this will need to be smaller at small desktop say 80px)
- Adds the grid system. Screenshots of this can be seen below. I used the existing `.grid` class for this which was previously used for other layouts such as cards but we won't be needing that here. 
- Adds the dark colour schemes for the Coral and Lagoon themes. I've set the default theme to be Coral dark.
- Adds a link to the design guide in the Figma file which could be useful for future reference.
- Adds variables for vertical spacing
- Removes colours and font weights that were added with kit but not needed.


### How to Test

The font can be seen on any component in the pattern library. To see the grid you'll need to add `<div class="grid"></div>` to a page and then inspect it for the time being. 

### Screenshots

<details><summary>Grid @ x-large, large and small breakpoints</summary>
<p>

![Screenshot 2024-01-05 at 10 08 57](https://github.com/torchbox/torchbox.com/assets/31627284/624fc3cb-6a7e-4fe0-b8a9-2b9651286b31)

![Screenshot 2024-01-05 at 10 09 30](https://github.com/torchbox/torchbox.com/assets/31627284/67579dd4-55dc-4a0b-bb49-589c53b82a23)

![Screenshot 2024-01-05 at 10 09 38](https://github.com/torchbox/torchbox.com/assets/31627284/eceb041f-2375-4736-afee-21d699e66c34)

</p>


### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability

- [ ] New font variants have not been added (they have but this is a one-off!)
